### ISSUE TITLE
Constantly getting: Error: The partial <partial_name> could not be found?

### ISSUE THREAD

**Issue by blaasvaer** [lazd/gulp-handlebars#83]
I can see my partials in my minified templates.js file and everything compiles nicely. But when I call the partial like this:

```

{{#each this}}
<ul id="{{code}}">
	<li>
		{{> thema_categories_level}}
	</li>
</ul>
{{/each}}

```
I keep getting: Error: The partial thema_categories_level could not be found

I'm following this guide:
https://github.com/lazd/gulp-handlebars/tree/master/examples/partials

But have spent the major part of the afternoon trying to get this to work (as I find myself doing a lot with handlebars : ( ).

Anyone has any idea what I'm missing ...

And yes I've got the file prefixed with underscore and all that. And it's there. In my file. But ...

**Comment by lazd** [comment]
Perhaps the naming is wrong on the partials. Inspect the output files and look for the `registerPartials` statements. If those are there with the right name and it still doesn't work, then it's a handlebars issue.

**Comment by blaasvaer** [comment]
This is getting weirder. I've scrapped my node_modules folder, and run a clean npm install. Now I get the classic:

`Error: Template was precompiled with an older version of Handlebars than the current runtime. Please update your precompiler to a newer version (>= 4.0.0) or downgrade your runtime to an older version (>= 2.0.0-beta.1).
`
I specifically point gulp-handlebars to require handlebars like this:

```
	.pipe(handlebars({
		handlebars: require('handlebars')
	}))

```
Which should point to the version in my package.json (right?), which is:

`    "handlebars": "^4.0.6"`

And I specifically point to the 4.0.6 runtime version.

Now, WHY is it compiled using version 2.0.0 beta!?

**Comment by blaasvaer** [comment]
OK, now I - just for fun(!) - pointed to the 3.0.3 version that comes with gulp-handlebars. And now it says:

`Error: Template was precompiled with a newer version of Handlebars than the current runtime. Please update your runtime to a newer version (>= 4.0.0).`

So obviously I'm somewhere in between 3.0.3 and 4.0.6. But how <swearing removed> am I to know what version is used for compiling!?

**Comment by blaasvaer** [comment]
OK, it turned out that: in an attempt to track down the bugger, I commented out my partial as in:

`
`
Now, that threw of the the compiler I pointed at (^4.0.6), and it ran home to mommy (which is obviously some unspecified version, probably not really).

I could see in the compiled templates, that one of the modules had:

`compiler:[6,">= 2.0.0-beta.1"]`

And when I manually changed the compiled and minified file it worked. So, back to tracking the commented out partials ... and the problem was solved.

So, don't comment out your partials (at least using markup tags).
